src/main/java/ladysnake/requiem/mixin/common/possession/possessed/PossessableLivingEntityMixin.java

```diff
@@ -335,7 +335,7 @@ private void onDeath(DamageSource deathCause, CallbackInfo ci) {
 
             if (possessor.isAlive() && secondLife != null) {    // player didn't get killed by attrition
                 possessor.world.spawnEntity(secondLife);
-                possessionComponent.stopPossessing(false);
+                possessionComponent.stopPossessing(world.getGameRules().getBoolean(RequiemGamerules.KEEP_ITEMS));
```

Comment:
This... doesn't seem right? It means when the gamerule is enabled, and you resurrect as a mob, you drop your items (instead of the current behavior where you always keep your items during resurrection).